docker/docker_tools_misc.py

```diff
@@ -88,7 +88,7 @@ def _start_servers(dev: bool) -> None:
     xqt(
         'sudo -u www-data env "PATH=$PATH" "REDIS_URI=$REDIS_URI" '
         "poetry run celery --app=scheduled_builder worker --pool=threads "
-        "--concurrency=3 --loglevel=info &",
+        "--concurrency=3 --loglevel=info --logfile=/dev/stdout &",
```

Comment:
celery probably will not get restarted very often but the redirect I used below `/proc/1/fd/1` ensures that the log output continues to go to the docker log even after the server is restarted.  Whereas a simple redirect to stdout does not.

Reply:
Thanks! I'll update that. Do you have any docs / links to discuss why this works? I'd like to include that if so. It's a neat technique.

Reply:
I found it while searching for a solution.  It is discussed in several places.  Such as https://github.com/moby/moby/issues/19616 and others.

Reply:
Great, thanks! I'll add that link.